fandogh_cli/fandogh_client/__init__.py

```diff
@@ -308,6 +311,16 @@ def deploy_manifest(manifest):
                              headers={'Authorization': 'JWT ' + token}
                              )
     if response.status_code != 200:
+        if response.status_code == 400:
+            keys = str(list(response.json().keys())[0]).split('.')
+            temp = x
+            for key in keys:
+                temp = temp.get(key, None)
+
+            if temp is not None:
```

Comment:
what if temp is none?

Reply:
If none -> normal error that has came from the server will be shown, like the old days :D